packages/server/src/middlewares/userInfo.ts

```diff
@@ -0,0 +1,24 @@
+import { Request, Response, NextFunction } from 'express';
+import jwt from 'jsonwebtoken';
+
+// interface AuthenticatedRequest extends Request {
+//   userID?: string;
+// }
+
+const userInfo = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const authHeader:any = req.headers.authorization;
+    if (!authHeader) {
+      return res.status(401).json({ message: 'Authorization header missing' });
+    }
+    const token = authHeader.split(' ')[1];
+
+    const { userID }: any = jwt.decode(token);
```

Comment:
Create a type called `JWTData` in your `types.ts` file. Use that type here instead of 'any'.
That way in line 17 you won't need to use `any` type as well.

Reply:
Done.